region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -503,13 +532,104 @@
           "canCarefulJump",
           "HiJump"
         ]},
-        {"obstaclesCleared": ["B"]},
-        {"or": [
-          "h_canUsePowerBombs",
-          {"obstaclesCleared": ["A"]}
-        ]}
+        {"obstaclesCleared": ["A", "B"]}
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Spark (Wall Jumps)",
+      "requires": [
+        {"obstaclesCleared": ["B", "E"]},
+        {"canShineCharge": {
+          "usedTiles": 32,
+          "gentleUpTiles": 6,
+          "openEnd": 1
+        }},
+        "canWalljump",
+        "canShinechargeMovementComplex",
+        {"shinespark": {"frames": 85}}
       ],
-      "clearsObstacles": ["A"]
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": [],
+          "useImplicitRequires": false
+        }
+      ]
+    },
+    {
+      "link": [2, 1],
+      "name": "Leave With Spark (Short Speedy Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B", "E"]},
+        {"canShineCharge": {
+          "usedTiles": 32,
+          "gentleDownTiles": 6,
+          "openEnd": 1
+        }},
+        "canShinechargeMovementComplex",
+        {"shinespark": {"frames": 57}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {
+          "position": "bottom"
+        }
+      },
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": [],
+          "useImplicitRequires": false
+        }
+      ]
```

Comment:
```suggestion
      ],
      "note": [
        "After storing a shinespark, start running from the left side of the vertical door and jump so that a shinespark at the apex of the jump will approximately align with the top of the far door."
      ]
```

Reply:
In this strat the PB blocks may be intact, so you wouldn't be able to start that far right. Was this note maybe intended for the "Long Speedy Jump" version?

Reply:
Yes sorry, this note would go with the long runway strat.

This short runway strat lines up well on its own but has some 🤔 aspects: jumping right near the end of the jump to have enough speed for jump height, and waiting until just before the shinespark frames expire.  But there are visual cues for both that are good enough.